src/raspbian/10/helix/arm32v6/Dockerfile

```diff
@@ -7,6 +7,7 @@ FROM scratch
 # But we can use a crossrootfs from our image generation scripts, and
 # extract into the image root
 
+RUN echo no-op
```

Comment:
`FROM scratch` containers (line 4) areliterally empty / zero-byte. The failures that `/bin/sh` not found is because docker's `RUN` command is trying to invoke `/bin/sh echo no-op` here and failing to find the executable.